unittests/InstructionCountCI/VEX_map1.json

```diff
@@ -134,16 +133,15 @@
       ]
     },
     "vmovsd xmm0, xmm1, xmm2": {
-      "ExpectedInstructionCount": 3,
+      "ExpectedInstructionCount": 2,
       "Optimal": "No",
```

Comment:
This is quite close to optimal. Since this is 128-bit result we can ZIP the results together to make this one instruction.